acme.go

```diff
@@ -238,6 +245,30 @@ func acceptChallenge(client *http.Client, config *Config, challenge Challenge) (
 	return chalResp, nil
 }
 
+// pollAuthorization checks the current status of an authorization request.
+//
+// If client argument is nil, DefaultClient will be used.
+func pollAuthorization(client *http.Client, config *Config, url string) (Authorization, error) {
+	if client == nil {
+		client = http.DefaultClient
+	}
+	res, err := client.Get(url)
```

Comment:
If you've got the URL for one of my authorization requests then yes, you can poll its status. You can't modify anything, though, since all non-polling requests are secure, but you could check how far along I am in authorizing my domain.